install/kubernetes/helm/istio/values.yaml

```diff
@@ -151,6 +151,16 @@ global:
   # will be exposed on an internal gateway
   meshExpansionILB: false
 
+  # Set to true to connect two kubernetes clusters using a LB gateway as the
+  # only entry point into the cluster (instead of requiring pod to pod
+  # connectivity across two clusters). Note that for this system to work,
+  # service objects from remote clusters have to be replicated to local
+  # cluster (without the pod selectors). In addition, endpoints have to be
+  # added to each service object pointing to the remote cluster's
+  # MCgatewayIP:443. All clusters should be using Istio mTLS and must have
+  # a shared root CA for this model to work.
```

Comment:
Would you mention that `istio-remote` will not needed in this solution? With this solution, each k8s cluster will have a standaline Istio control plane.